staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -3929,7 +3929,7 @@ type PodResourceClaimStatus struct {
 	Name string `json:"name" protobuf:"bytes,1,name=name"`
 
 	// ResourceClaimName is the name of the ResourceClaim that was
-	// generated for the Pod in the namespace of the Pod. It this is
+	// generated for the Pod in the namespace of the Pod. If this is
```

Comment:
This should be changed in the .proto file and .go files should be re-generated.

Reply:
No, for the Kubernetes API, the `types.go` files are the source. One has to run various update scripts to propagate this fix to other generated files.

@Monokaix: this is not worth the time that we would have to spend on getting API approval. Can you take out this part? I'll add it to a [pending API update PR](https://github.com/kubernetes/kubernetes/pull/124075).

Reply:
It's good idea, have removed that: ) @pohly